### ISSUE TITLE
example of dynamic web page with input and output

### ISSUE THREAD

**Issue by ghislainborremans** [gmag11/FSBrowserNG#28]
Hello 

i keep fighting with this. 
Is it possible to include an example of a dynamic webpage with the coding to put the data on that page. 
I want to enter 2 values (calibration) and to show the temperature of 2 sensors in 1 page. 
It should be simple but i am not getting there. 

Any help would be aprreciated;

**Comment by species5618** [comment]
It the one exmaple i have not had time to get to 

i know how i want to do it, i will get to it soon

**Comment by species5618** [comment]
call back example now include miniajax get and form post example for dynamic user content

updated data folder to include user.html example

also added ability to store user config values in  SPIFFs in userconfig.json file

sample code
			ESPHTTPServer.save_user_config("Test", "Test");
			String T = "";
			ESPHTTPServer.load_user_config("Test", T);
			Serial.print("S: ");
			Serial.println(T);
		
			//ESPHTTPServer.save_user_config("TestI", 10);
			int I = 0;
			ESPHTTPServer.load_user_config("TestI", I);
			Serial.print("I: ");
			Serial.println(I);

			//ESPHTTPServer.save_user_config("TestF", 10);
			float F = 0.0;
			ESPHTTPServer.load_user_config("TestF", F);
			Serial.print("F: ");
			Serial.println(F);

**Comment by species5618** [comment]
callback example updated to save the data in the SPIFFS userconfig.json file

**Comment by gmag11** [comment]
Several pages on the project show how this is done. You can find it on https://github.com/gmag11/FSBrowser/blob/master/DynamicData.cpp file. It uses a simple implementation of AJAX.

**Comment by ghislainborremans** [comment]
thank you. I will try it when i am back and return some feedback..

**Comment by mancior** [comment]
Hello
This is really great project! Thank You for it.
There are samples  of callback with POST and REST. Please give some with JSON, please.

I have json structure {"items":[{"addr":"10380865","name":"test1"},{"addr":"1415368","name":"test2"},{"addr":"5672520","name":"test3"}]} which I got after using
` console.log('currentData to write:' , JSON.stringify(currentData));` 
But how to write it to file and how to read it again from that file? Thank You in advance.

**Comment by ghislainborremans** [comment]
Hello
I have been able to use the "callback" example. this is very nice. 
However i do not understand that each time both the REST and Post callback are executed. 
is this normal? and if so why ?

**Comment by species5618** [comment]
no, each callback should only go off once if the folder relevant folder is called
i cant see anything obvious in the code
i dont have access to hardware for a few days i will check at the weekend

**Comment by species5618** [comment]
can ask how you discovered /think this is the case

**Comment by species5618** [comment]
sorry just re reading you question

the form example, will use REST to read the data on page load
and then when you click save will "post" to save the data
and then refreshes, so will call call REST to get the new data
